packages/commerce-sdk-react/src/hooks/types.ts

```diff
@@ -54,16 +54,66 @@ export type ShopperSearchInstance = ShopperSearch<
 >
 
 export interface ApiClients {
-    shopperBaskets: ShopperBasketsInstance
-    shopperContexts: ShopperContextsInstance
-    shopperCustomers: ShopperCustomersInstance
-    shopperDiscoverySearch: ShopperDiscoverySearchInstance
-    shopperGiftCertificates: ShopperGiftCertificatesInstance
-    shopperLogin: ShopperLoginInstance
-    shopperOrders: ShopperOrdersInstance
-    shopperProducts: ShopperProductsInstance
-    shopperPromotions: ShopperPromotionsInstance
-    shopperSearch: ShopperSearchInstance
+    shopperBaskets: ShopperBaskets<{
+        clientId: string
+        organizationId: string
+        shortCode: string
+        siteId: string
+    }>
```

Comment:
Minor, this type could be extracted to 
```suggestion
    shopperBaskets: ShopperBaskets<CommerceApiProviderParams>
```
Minor DRY. Could possibly be derived from `CommerceApiProviderProps`, but might take some rekerjiggering to avoid circular deps.

Reply:
Sorry, I should've been more DRY. I've refactored the types in the latest commit.